solvers/optimization_utils.h

```diff
@@ -0,0 +1,60 @@
+#include "drake/solvers/mathematical_program.h"
```

Comment:
Sorry that I keep adding new comments after the approval. 
I think we need `#pragma once` here. Otherwise, we would re-define the default parameters such as eps in SecondOrderCost, and this would give compile error.

Reply:
Done.